src/components/AdminDashboard/AdminDashboard.js

```diff
@@ -70,117 +68,76 @@ const AdminDashboard = () => {
+      .catch(err => console.log(err));
 
-  // getting approved incidents from database
-  useEffect(() => {
-    oktaAxios
-      .get('/dashboard/incidents/approved')
-      .then(res => {
-        setIncidents(res.data);
-      })
-      .catch(err => {
-        console.log(err);
-      });
-  }, []);
+    getPendingIncidents(oktaAxios)
+      .then(res => setPendingIncidents(res))
+      .catch(err => console.log(err));
 
-  // getting form-responses from DS database
+    getFormResponses(oktaAxios)
+      .then(res => setFormResponses(res))
+      .catch(err => console.log(err));
+  };
+
+  // getting all incident data
   useEffect(() => {
-    axios
-      .get(
-        'http://hrf-bw-labs37-dev.eba-hz3uh94j.us-east-1.elasticbeanstalk.com/to-approve'
-      )
-      .then(res => {
-        setFormResponses(res.data);
-      })
-      .catch(err => {
-        console.log(err);
-      });
+    fetchIncidents();
   }, []);
 
-  // setting up pagination display on dashboard
-  useEffect(() => {
-    const start = incidentsPerPage * pageNumber - incidentsPerPage;
-    const newCurrentSet = unapprovedIncidents.slice(
-      start,
-      start + incidentsPerPage
-    );
-    setCurrentSet(newCurrentSet);
-  }, [pageNumber, incidentsPerPage, unapprovedIncidents]);
-
-  // incident selection (checkbox) functions
-  const selectAll = () => {
-    setAllSelected(!allSelected);
-    if (!allSelected) {
-      setSelected(currentSet.map(data => data.incident_id));
-    } else {
-      setSelected([]);
+  const approveAndRejectHandler = async (newStatus) => {
+    const currentList = getCurrentList();
+    const { selected, source } = splitIncidentsByIds(currentList, selectedIds);
+
+    // setting lat and long for approved incidents
+    if (newStatus === 'approved') {
+      for (const inc of selected) {
+        if (inc.city && inc.state) {
+          const latAndLong = await getLatAndLong(inc);
+          inc.lat = latAndLong[0];
+          inc.long = latAndLong[1];
+        }
+      }
     }
-  };
 
-  const approveAndRejectHandler = async evt => {
-    evt.preventDefault();
-    const [reviewedIncidents, unreviewedIncidents] = sortApproved(
-      currList,
-      selected
-    );
-    // MVP: One-by-one approvals. Needs work for multiple approvals simultaneously
-    const latAndLong = await getLatAndLong(reviewedIncidents[0]);
-    reviewedIncidents[0].lat = latAndLong[0];
-    reviewedIncidents[0].long = latAndLong[1];
-
-    putIncidents(oktaAxios, reviewedIncidents, confirmStatus);
-
-    if (listType === 'unapproved') {
-      setUnapprovedIncidents(unreviewedIncidents);
-    } else if (listType === 'form-responses') {
-      //did 'else if' incase we wanted to add Reject button to Approved list
-      setFormResponses(unreviewedIncidents);
-    }
-    setAllSelected(false);
-    setSelected([]);
-    setConfirmApprove(false);
-    setConfirmReject(false);
-    if (pageNumber > lastPage) {
-      setPageNumber(pageNumber - 1);
-    }
-  };
```

Comment:
Somewhat confused by why we're removing lines 145-157 here.

Reply:
They're actually still there, lines 121-131.